addons/account/tests/test_sequence_mixin.py

```diff
@@ -118,6 +118,35 @@ def test_sequence_change_date_with_quick_edit_mode(self):
             invoice_form.date = '2017-01-01'
             self.assertEqual(invoice_form.name, 'INV/2017/00001')
 
+    def test_sequence_empty_change_date_with_quick_edit_mode(self):
+        """ Ensure the names of all but the first moves in a period are empty in quick edit mode """
+        self.env.company.quick_edit_mode = 'in_invoices'
+
+        bill_1 = self.env['account.move'].create({
+            'partner_id': 1,
+            'move_type': 'in_invoice',
+            'date': '2016-01-01',
+            'line_ids': [
+                Command.create({
+                    'name': 'line',
+                    'account_id': self.company_data['default_account_revenue'].id,
+                }),
+            ]
+        })
+        # First move in a period gets a name
+        self.assertEqual(bill_1.name, 'BILL/2016/01/0001')
+
+        bill_2 = bill_1.copy({'date': '2016-01-02'})
+        # Subsequent moves in the same period get no name in draft mode
+        self.assertFalse(bill_2.name)
+
+        bill_3 = bill_1.copy({'date': '2016-01-03'})
+        bill_4 = bill_1.copy({'date': '2016-01-04'})
+        (bill_3 + bill_4).date = fields.Date.from_string('2016-02-01')
+        # Same works with updating multiple moves
+        self.assertEqual(bill_3.name, 'BILL/2016/02/0001')
+        self.assertFalse(bill_4.name)
```

Comment:
Check that the field is editable using a form emulator since this is your goal.
The fact that it is falsy is only an implementation detail